Helper/Price.php

```diff
@@ -379,16 +390,30 @@ private function getConfigurableProductPrice(Product $product, $finalPrice, $inc
         if (count($skuPrices) === 0) {
             return $price;
         }
-        $priceColumn = array_column($skuPrices, "final_price");
+        $priceColumn = array_column($skuPrices, self::KEY_SKU_FINAL_PRICE);
         array_multisort($priceColumn, SORT_ASC, $skuPrices);
         $minSku = reset($skuPrices);
-        if ($finalPrice && isset($minSku['final_price'])) {
-            $price = $minSku['final_price'];
-        } elseif (isset($minSku['final_price'])) {
-            $price = $minSku['price'];
+        if ($finalPrice && isset($minSku[self::KEY_SKU_FINAL_PRICE])) {
+            $price = $minSku[self::KEY_SKU_FINAL_PRICE];
+        } elseif (isset($minSku[self::KEY_SKU_PRICE])) {
```

Comment:
Price or final price here?

Reply:
It's correct. Final price should be used only if the `$finalPrice` is true